include/sortcheck.h

```diff
@@ -199,6 +199,58 @@ inline bool binary_search_checked_full(_ForwardIterator __first,
   return binary_search_checked_full(__first, __last, __val, compare, file, line);
 }
 
+template<typename _ForwardIterator, typename _Tp, typename _Compare>
+inline _ForwardIterator lower_bound_checked(_ForwardIterator __first,
+                                  _ForwardIterator __last,
+                                  const _Tp &__val, _Compare __comp,
+                                  const char *file, int line) {
+  // Ordered
+  if (__first != __last) {
+    bool is_prev_less = true;
+    unsigned pos = 0;
+    for (_ForwardIterator it = __first; it != __last; ++it, ++pos) {
+      bool is_less = *it < __val;
```

Comment:
`__comp(*it, __val)`? Would you mind fixing this for `binary_search_checked` as well?

Reply:
Yes, it is a copy paste error
It seems, it is already fixed for `binary_search_checked`